packages/app/src/public-types.ts

```diff
@@ -139,6 +182,66 @@ export interface FirebaseAppSettings {
   automaticDataCollectionEnabled?: boolean;
 }
 
+/**
+ * @public
+ *
+ * Configuration options given to {@link (initializeServerApp:1) | initializeServerApp()}
+ */
+export interface FirebaseServerAppSettings extends FirebaseAppSettings {
+  /**
+   * An optional Auth ID token used to resume a signed in user session from a client
+   * runtime environment.
+   *
+   * If provided, the FirebaseServerApp instance will work to validate the token even before Auth
+   * is initialized. The result of the validation can be queried via by the application by invoking
+   * {@link (FirebaseServerApp.authIdTokenVerified()}. Awaiting the Promise returned by
+   * {@link (FirebaseServerApp.authIdTokenVerified()} is highly recommended if an authIdToken token
+   * is provided.
+   *
+   * Invoking getAuth() with a FirebaseServerApp configured with a validated authIdToken will cause
+   * an automatic attempt to sign in the user that the authIdToken represents. The token
+   * needs to have been recently minted for this operation to succeed, otherwise it will fail
+   * validation.
+   *
+   * If the token fails local verification, or if the Auth service has deemed it invalid when
+   * the Auth SDK is initialized, then a warning is logged to the console and the Auth SDK will not
+   * sign in a user upon initalization.
+   *
+   * If a user is successfully signed-in, then the Auth instance's onAuthStateChanged callback
+   * will be invoked with the User as per standard Auth flows. However, users created via
+   * authIdTokens do not have a refresh token and any attempted refresh operation will fail.
+   */
+  authIdToken?: string;
+
+  /**
+   * An optional object. If provided, the Firebase SDK will use a FinalizationRegistry
+   * object to monitor the Garbage Collection status of the provided object, and the
+   * Firebase SDK will release its refrence on the FirebaseServerApp instance when the
+   * provided object is garbage collected.
+   *
+   * The intent of this field is to help reduce memory overhead for long-running cloud
```

Comment:
Could "long-running cloud functions." just be "functions?"

(Cloud Functions is a product name).

Reply:
Done.